runtime-manager/Makefile

```diff
@@ -157,6 +157,18 @@ runtime_manager.eif: target/x86_64-unknown-linux-musl/release/runtime_manager_en
 target/x86_64-unknown-linux-musl/release/runtime_manager_enclave: Cargo.toml $(Nitro_Src)
 	cargo build --target x86_64-unknown-linux-musl --release --features nitro
 
+####### Linux (AArch64 and X64) processes  #########
+Linux_Src = $(COMMON_Src) src/runtime_manager_linux.rs src/main.rs
+
+linux: Cargo.toml $(Linux_Src)
+	cargo build --release --features linux
+
+linux-x64: Cargo.toml $(Linux_Src)
+	cargo build --target x86_64-unknown-linux-gnu --release --features linux
+
+linux-aarch64: Cargo.toml $(Linux_Src)
+	cargo build --target aarch64-unknown-linux-gnu --release --features linux
```

Comment:
I'm curious why we have explicit `x64` and `aarch64` targets. Is it not sufficient to assume we're compiling for the native target?

If the goal is to be able to cross compile for testing/CI, it may be more scalable to use an environment variable:

``` make
LINUX_TARGET_FLAG := $(if $(LINUX_TARGET),--target $(LINUX_TARGET))

linux-aarch64: Cargo.toml $(Linux_Src)
    cargo build $(LINUX_TARGET_FLAG) --release --features linux
```

At build time:

``` bash
$ make linux LINUX_TARGET=x86_64-unknown-linux-gnu
```

(Note I have not tested this)